src/Nox.Core/Domain/IRepository.cs

```diff
@@ -6,17 +6,27 @@ namespace Nox.Domain;
 public interface IRepository
 {
     IQueryable<T> Query<T>(params Expression<Func<T, object>>[] includeExpressions) where T : class, IEntity;
-    
     ValueTask<T?> FindAsync<T>(params object?[]? keyValues) where T : class, IEntity;
     ValueTask<T?> FindAsync<T>(object?[]? keyValues, CancellationToken cancellationToken) where T : class, IEntity;
-    
     ValueTask<T?> FindAndIncludeAsync<T>(object?[]? keyValues, Expression<Func<T, IEnumerable<object>>> includeExpression, CancellationToken cancellationToken) where T : class, IEntity;
     ValueTask<T?> FindAndIncludeAsync<T>(object?[]? keyValues, Expression<Func<T, object?>> includeExpression, CancellationToken cancellationToken) where T : class, IEntity;
-
+    ValueTask<T?> FindAndIncludeAsync<T, TProperty, TChildProperty>(object?[]? keyValues,
```

Comment:
including localized should be unnecessary. This is being setup to cascade delete
we want to "hide" the localized concept from our domain.

For example when deleting a single owned entity we are not doing it, so the related localized is being deleted

see NoxDatabaseConfigurator for Localized Entity

Reply:
Desired changes have been applied